composer.json

```diff
@@ -10,22 +10,22 @@
     "require": {
         "php": "^7.1",
         "cpliakas/git-wrapper": "^1.7",
-        "doctrine/inflector": "^1.1",
-        "knplabs/github-api": "^2.5",
-        "knplabs/packagist-api": "^1.3",
+        "doctrine/inflector": "^1.2",
+        "knplabs/github-api": "^2.6",
+        "knplabs/packagist-api": "^1.5",
         "maknz/slack": "^1.7",
         "php-http/guzzle6-adapter": "^1.1",
-        "sebastian/diff": "^1.4",
-        "silex/silex": "^2.0",
-        "sllh/php-cs-fixer-styleci-bridge": "^2.0",
-        "symfony/config": "^3.0",
-        "symfony/console": "^3.0",
-        "symfony/filesystem": "^3.0",
-        "twig/twig": "^1.24",
-        "vlucas/phpdotenv": "^2.2"
+        "sebastian/diff": "^2.0",
+        "silex/silex": "^2.2",
+        "symfony/config": "^3.3",
```

Comment:
Why those changes? They are not part of the PR, please revert and just keep `php-cs-fixer-styleci-bridge` removal.